src/styles/blocks/card-block/content-card.scss

```diff
@@ -0,0 +1,134 @@
+.content {
+  margin: 0 auto;
+  display: grid;
+  // gap: 48px;
+  column-gap: 48px;
+  row-gap: 46px;
+  max-width: 944px;
+  padding-top: 50px;
+
+  @media (max-width:488px) {
+    grid-template-columns: 1fr;
+  }
+
+  @media (min-width:488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width:768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width:1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+  &__card {
+    width: 200px;
+    border: 1px solid #f3f3f3;
+    background-color: $background_white-color;
+    border-radius: 5px;
+    padding: 32px 16px 16px;
+
+    &-mac-image {
+      display: block;
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
+
+    &-title {
+      margin-top: 40px;
+      display: block;
+    }
+
+    &-name {
+      font-weight: 500;
+      font-size: 12px;
+      color: $dark_blue-color-text;
+      line-height: 18px;
+    }
+
+    &-code {
+      margin-top: 4px;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $shadow_grey_small;
+    }
+
+    &-stats {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 16px;
+    }
+
+    &-review {
+      font-weight: 400;
+      font-size: 10px;
+      color: $dark_blue-color-text;
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-top: 24px;
+    }
+
+    &-price-name {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: $shadow_grey_small;
+    }
+
+    &-price-cost {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $dark_blue-color-text;
+    }
+
+    &-btn {
+      display: block;
+      height: 40px;
+      text-align: center;
+      margin-top: 16px;
+      width: 166px;
+      padding: 11px 0;
+      border-radius: 5px;
+      border: 1px solid transparent;
+      text-transform: uppercase;
+      text-decoration: none;
+      font-weight: 700;
+      font-size: 14px;
+      background-color: $main-blue-color;
+      color: $background_white-color;
+      cursor: pointer;
+    }
+
+    &-btn:hover {
+      background-color: $background_white-color;
+      color: $main-blue-color;
+      border-color: $main-blue-color;
+    }
+
+  }
+  @media (max-width:1200px) {
+    &--position {
```

Comment:
The use of the modifier '--position' is unusual. In BEM, modifiers usually describe a state or a version of the block or element. Here, it looks like you are adjusting the margin, which could probably be added directly to the block or element that requires it.